### PR TITLE
feat(cluster): Implement `CLUSTER SLOTS`.

### LINKED ISSUE #1278: Implement `CLUSTER NODES` command

### PR DESCRIPTION
**Pull request by chakaz**
Works in all 3 modes: emulated, cluster and unconfigured-cluster.

Fixes #1278

### CODE REVIEW

File: src/server/cluster/cluster_family.cc [hunk 1]
```diff
@@ -35,15 +35,7 @@ using namespace std;
...
-void BuildClusterSlotNetworkInfo(ConnectionContext* cntx, std::string_view host, uint32_t port,
-                                 std::string_view id) {
-  constexpr unsigned int kNetworkInfoSize = 3;
-
-  (*cntx)->StartArray(kNetworkInfoSize);
-  (*cntx)->SendBulkString(host);
-  (*cntx)->SendLong(port);
-  (*cntx)->SendBulkString(id);
-}
+constexpr string_view kClusterNotConfigured = "-Cluster is not yet configured";
```
Comment (adiholden) [inline src/server/cluster/cluster_family.cc:38]: why did you add the '-' char at the beginning?

**Review by adiholden — APPROVED** [review]

File: src/server/cluster/cluster_family.cc (hunk 1 above)
Comment (chakaz) [reply src/server/cluster/cluster_family.cc]: I should not have, sorry!

**Review by adiholden — APPROVED** [review]
